guides/common/modules/proc_creating-a-user.adoc

```diff
@@ -4,31 +4,30 @@
 Use this procedure to create a user.
 To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-a-user_{context}[].
 
-If a new user is not assigned to an organization, their access is limited.
-To grant systems rights to users, assign them to an organization.
-The next time the user logs on to {Project}, the user's account has the correct system rights.
-
 .Procedure
 . In the {ProjectWebUI}, navigate to *Administer* > *Users*.
 . Click *Create User*.
-. In the *Login* field, enter a username for the user.
-. In the *Firstname* and *Lastname* fields, enter the real first name and last name of the user.
-. In the *Mail* field, enter the user’s email address.
-. In the *Description* field, add a description of the new user.
-. Select a specific language for the user from the *Language* list.
-. Select a timezone for the user from the *Timezone* list.
-+
-By default, {ProjectServer} uses the language and timezone settings of the user’s browser.
-. Set a password for the user:
-.. From the *Authorized by* list, select the source by which the user is authenticated.
-** *INTERNAL*: to enable the user to be managed inside {ProjectServer}.
-** *EXTERNAL*: to configure external authentication as described in {InstallingServerDocURL}Configuring_External_Authentication_{project-context}[Configuring External Authentication] in _{InstallingServerDocTitle}_.
-.. Enter an initial password for the user in the *Password* field and the *Verify* field.
+. Enter the account details for the new user.
 . Click *Submit* to create the user.
 
+For example, you can specify the following user account details:
```

Comment:
"can" and line 15 "must" sounds weird to me.

I'm also unsure about starting this sentence with "For example,". Maybe like this instead?

```suggestion
.User account details
```

Reply:
I don't really require additional heading, but I would like to somehow remove the for example part if possible. Just to emphasize that there are no example values but instead description of a few fields.